src/Symfony/Component/Debug/Debug.php

```diff
@@ -53,6 +53,7 @@ public static function enable($errorReportingLevel = null, $displayErrors = true
             ini_set('display_errors', 1);
         }
         $handler = ErrorHandler::register();
+        $handler->setDefaultLogger(new BootstrappingLogger());
```

Comment:
you're right, one logger per level is required. For removing the bootstrapping logger, there is one: $handler->setLoggers with `null`. We could create a helper, but I doubt anyone will use it so I'd say no need for more.